server/src/domain/access/access.core.ts

```diff
@@ -139,8 +139,9 @@ export class AccessCore {
       // uses album id
       case Permission.ACTIVITY_CREATE:
         return (
-          (await this.repository.album.hasOwnerAccess(authUser.id, id)) ||
-          (await this.repository.album.hasSharedAlbumAccess(authUser.id, id))
+          ((await this.repository.album.hasOwnerAccess(authUser.id, id)) ||
+            (await this.repository.album.hasSharedAlbumAccess(authUser.id, id))) &&
+          (await this.repository.album.hasActivityAccess(authUser.id, id))
```

Comment:
Hmm, I don't think we want this. We can just search for an activity enabled album in the access implementation.